### PR TITLE
integration: Fix NetworkManager preflight messages

### PR DESCRIPTION
**Pull request by cfergeau**
The systemd changes also modified the messages related to
systemd/NetworkManager interaction. Hopefully this was the last missing
piece and integration tests will be back to green.

### CODE REVIEW

**Review by guillaumerose — APPROVED** [review]

**Comment by praveenkumar** [comment]
let's wait for the CI before merge, we will only merge when it's green.

**Comment by cfergeau** [comment]
> let's wait for the CI before merge, we will only merge when it's green.

Agreed, exactly what I had in mind :+1:

**Comment by cfergeau** [comment]
CI  failed but I don't think it's related to the systemd changes
```
 Command stderr: level=error msg="Error approving the node csr: Not able to get csr names (ssh command error:\ncommand : oc get csr -ojson --context admin --cluster crc --kubeconfig /opt/kubeconfig\nerr     : exit status 1\noutput  : {\n    \"apiVersion\": \"v1\",\n    \"items\": [],\n    \"kind\": \"List\",\n    \"metadata\": {\n        \"resourceVersion\": \"\",\n        \"selfLink\": \"\"\n    }\n}\nThe connection to the server api.crc.testing:6443 was refused - did you specify the right host or port?\n : )"
Error approving the node csr: Not able to get csr names (ssh command error:
command : oc get csr -ojson --context admin --cluster crc --kubeconfig /opt/kubeconfig
err     : exit status 1
output  : {
    "apiVersion": "v1",
    "items": [],
    "kind": "List",
    "metadata": {
        "resourceVersion": "",
        "selfLink": ""
    }
}
The connection to the server api.crc.testing:6443 was refused - did you specify the right host or port?
 : )
```

**Comment by cfergeau** [comment]
CI went through this time (except for centos which is not unusual to say the least ;), I'll merge this.

**Pull request merged**
